script/build-ubuntu.sh

```diff
@@ -20,15 +20,33 @@ if [[ -z "${CURL_INSTALL_DIR}" ]]; then
   exit 1
 fi
 
-if [ "$TARGET_ARCH" = "64" ]; then
-  DEPENDENCY_ARCH="amd64"
-else
-  DEPENDENCY_ARCH="x86"
```

Comment:
I don't see a replacement for x86 🤔

Reply:
refer to github releases: https://github.com/desktop/dugite-native/releases/tag/v2.39.2 there are no x86 builds being made as of now

Reply:
your CI and scripting did not even function on x86 before this PR either. if you look at this line from before you will see that you downloaded amd64 regardless of architecture https://github.com/desktop/dugite-native/blob/d2144bef672f1dd58e25634b5d61867553549169/script/build-ubuntu.sh#L75

if you want x86, that is a NEW feature and will also require cross compilation similar to armhf/arm64.

Reply:
Ohhh, sorry, I thought we included it. I see you've added it now. It wasn't necessary, but that's ok 😅  Thanks!